uPortal-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -90,15 +90,15 @@
         <filter-class>org.jasig.cas.client.validation.Cas20ProxyReceivingTicketValidationFilter</filter-class>
         <init-param>
           <param-name>casServerUrlPrefix</param-name>
-          <param-value>${environment.build.cas.protocol}://${environment.build.cas.server}${environment.build.cas.context}</param-value>
+          <param-value>http://localhost:8080/cas</param-value>
```

Comment:
Did we move to gradle too quickly? Because filtering seems like it is the main issue we have been running into lately and gradle does not help us in any way solve this problem. I think it is worth waiting on merging this until we can get the JVM args side of things figured out so it is not a problem.